tests/GeneralStateTestsRunner.js

```diff
@@ -72,6 +72,7 @@ function runTestCase (testData, t, cb) {
 }
 
 module.exports = function runStateTest (options, testData, t, cb) {
+  debugger
```

Comment:
Is this some new ES* magic?

Reply:
since `testState` is being run with `node inspect` this acts as a breakpoint for the debugger.